src/presence_analyzer/templates/mean_time_weekday.html

```diff
@@ -0,0 +1,15 @@
+{% extends 'main.html' %}
+
+{% block script %}
+    <script type="text/javascript" src="/static/js/mean_time_weekday.js"></script>
+{% endblock %}
+
+{% block links %}
```

Comment:
We're repeating this on every template, move links to base as they should be same on every sub-page.